commonjs/frontend-form/field/recaptcha.js

```diff
@@ -1,13 +1,28 @@
 var fieldRegistry = require('kwf/frontend-form/field-registry');
 var Field = require('kwf/frontend-form/field/field');
 var kwfExtend = require('kwf/extend');
+var $ = require('jQuery');
 
-Recaptcha = kwfExtend(Field, {
+
+var Recaptcha = kwfExtend(Field, {
+    initField: function() {
+        var cb = ("kwfUp-recaptchaCallback" + this.getFieldName()).replace(/(_|\-)/g, '');
+        window[cb] = (function() {
+            window.grecaptcha.render(this.getField(), {
+                'sitekey' : this.getField().getAttribute('data-site-key'),
+                'callback' : (function(response) {
+                    this.form.errorStyle.hideFieldError(this);
+                }).bind(this)
+            });
+        }).bind(this);
+        var url = 'https://www.google.com/recaptcha/api.js?onload=' + cb + '&render=explicit';
+        $.getScript(url);
```

Comment:
This loads the script using XHR. Any reason for that?

And does this work for multiple fields on one page?